packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -715,18 +715,15 @@ export default class TableColumnResizeEditing extends Plugin {
 	 * @private
 	 */
 	_registerResizerInserter() {
-		const view = this.editor.editing.view;
-
-		view.on( 'render', () => {
-			for ( const item of view.createRangeIn( view.document.getRoot() ) ) {
-				if ( ![ 'td', 'th' ].includes( item.item.name ) ) {
-					continue;
-				}
-
-				view.change( viewWriter => {
-					ensureColumnResizerElement( viewWriter, item.item );
-				} );
-			}
-		}, { priority: 'lowest' } );
+		this.editor.conversion.for( 'editingDowncast' ).add( dispatcher => {
+			dispatcher.on( 'insert:tableCell', ( evt, data, conversionApi ) => {
+				const mapper = conversionApi.mapper;
```

Comment:
Just a nitpick - I'm not a great fan of creating constants that only skip a level of scope and are only used once, with unchanged name even. I'd rather have `const viewElement = conversionApi.mapper.toViewElement( modelElement );` and one `const` less.

`modelElement` is similar but it at least adds some semantics, so I don't have a problem with that.